testing/mock.ts

```diff
@@ -1082,15 +1097,15 @@ export function stub<
   func?: (this: Self, ...args: Args) => Return,
 ): Stub<Self, Args, Return> {
   if (self[property] !== undefined && typeof self[property] !== "function") {
-    throw new MockError("Property is not an instance method");
+    throw new TypeError("Property is not an instance method");
   }
   if (isSpy(self[property])) {
     throw new MockError("Already spying on instance method");
   }
 
   const propertyDescriptor = Object.getOwnPropertyDescriptor(self, property);
   if (propertyDescriptor && !propertyDescriptor.configurable) {
-    throw new MockError("Cannot spy on non-configurable instance method");
+    throw new TypeError("Cannot spy on non-configurable instance method");
```

Comment:
Ditto. This looks like an error on attempting invalid stubbing. Why isn't this MockError?

Reply:
Another point. The error message probably should use the verb `stub`?
```suggestion
    throw new TypeError("Cannot stub on non-configurable instance method");
```

Reply:
Let me address the error message in a separate PR.